src/components/QuoteRequest/Groomer/GroomerShopRequests.jsx

```
@@ -6,17 +6,18 @@ import useAuthStore from "@/store/authStore";
 import Modal from "@/components/common/modal/modal";
 import { RequestReject } from "@/queries/quoteRequestQuery";
 
-function GroomerShopRequests({ Infos }) {
+function GroomerShopRequests({ Infos, onRequestReject }) {
+  if (Infos == null) return <></>;
   return (
     <>
       {Infos.map((Info) => {
-        return <GroomerEstimate Info={Info} />;
+        return <GroomerEstimate Info={Info} onRequestReject={onRequestReject} />;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)